### PR TITLE
feat: implement venue and instruction entities

### PR DESCRIPTION
**Pull request by monitz87**

### CODE REVIEW

**Review by VictorVicente — APPROVED** [review]

File: src/api/entities/Instruction/index.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+   * Check if a value is of type [[UniqueIdentifiers]]
+   */
+  public static isUniqueIdentifiers(identifier: unknown): identifier is UniqueIdentifiers {
+    const { id } = identifier as UniqueIdentifiers;
+
+    return id instanceof BigNumber;
+  }
+
+  /**
+   * Identifier number of the venue
```
Comment (shuffledex) [inline src/api/entities/Instruction/index.ts:36]: ...of the instruction

**Pull request merged**